style.css

```diff
@@ -224,7 +224,6 @@ span {
   position: fixed;
   top: 50%;
   left: 50%;
-  width: 800px;
   display: none;
   transition: transform 0.4s, top 0.4s;
   z-index: 200;
```

Comment:
I have also a few friendly styling suggestions to make your popup even closer to the given design:

- [ ]  Let's remove the background color, as it doesn't quite match the lovely [Figma design provided](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&mode=dev).

- [ ] Could you please use the same font size for your title? It will help maintain consistency.

- [ ] It would be fantastic if you could check the size of your project image and try to make it closer to the Figma template.

- [ ] Please, don't forget to update your buttons to match the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&mode=dev). It'll look even more professional.

- [ ] By the way, your badges look great, but let's make them a bit smaller to match the [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&mode=dev). Also, the text inside should have the correct font weight and size for a perfect look.

**_Don't forget to apply these changes to your mobile version as well._**

   ### Your design:
   ![image](https://github.com/RolandAmini/portfolio-mobile/assets/107952724/49a0c3f7-bcff-48bf-9948-9f7875a00bc1)


   ### Figma design:
   ![image](https://github.com/RolandAmini/portfolio-mobile/assets/107952724/89c5279a-4a56-4ed8-80eb-f24e0de176c1)